box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxEntity.java

```diff
@@ -111,6 +111,10 @@ public static BoxEntity createEntityFromJson(final String json){
             BoxFileVersion version = new BoxFileVersion();
             version.createFromJson(json);
             return version;
+        } else if (createdByEntity.getType().equals(BoxMetadata.TYPE)) {
```

Comment:
Let's remove this if it's not needed as I don't think BoxMetadata should extend from BoxEntity since it doesn't have an id or type